src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/BeneficialOwnerChangeServiceTest.java

```diff
@@ -0,0 +1,333 @@
+        beneficialOwnerIndividualDto);
+
+    assertNotNull(result);
+    assertEquals(new PersonName("John", ""), result.getPsc().getPersonName());
+  }
+
+  @Test
+  void testCovertBeneficialOwnerIndividualToChangeLastNameOnly() {
+    BeneficialOwnerIndividualDto beneficialOwnerIndividualDto = new BeneficialOwnerIndividualDto();
+    beneficialOwnerIndividualDto.setChipsReference("1234567890");
+    beneficialOwnerIndividualDto.setLastName("Doe");
+    beneficialOwnerIndividualDto.setNonLegalFirmMembersNatureOfControlTypes(List.of(
+        NatureOfControlType.SIGNIFICANT_INFLUENCE_OR_CONTROL));
+
+    when(mockPublicPrivateBoPair.getLeft().getName()).thenReturn("Jane Doe");
+
+    Map<String, Pair<PscApi, PrivateBoDataApi>> publicPrivateBoMap = new HashMap<>();
+    publicPrivateBoMap.put(beneficialOwnerIndividualDto.getChipsReference(),
+        mockPublicPrivateBoPair);
+
+    when(mockPublicPrivateBoPair.getLeft().getNationality()).thenReturn("British,French");
+    when(mockPublicPrivateBoPair.getLeft().getNaturesOfControl()).thenReturn(
+        new String[]{"OE_OWNERSHIPOFSHARES_MORETHAN25PERCENT_AS_TRUST"});
+    when(publicPrivateBo.get(beneficialOwnerIndividualDto.getChipsReference())).thenReturn(
+        mockPublicPrivateBoPair);
+
+    var result = beneficialOwnerChangeService.covertBeneficialOwnerIndividualToChange(
+        beneficialOwnerIndividualDto);
+
+    assertNotNull(result);
+    assertEquals(new PersonName("", "Doe"), result.getPsc().getPersonName());
+  }
+
+  @Test
+  void testCovertBeneficialOwnerIndividualToChangeNoChange() {
+    BeneficialOwnerIndividualDto beneficialOwnerIndividualDto = new BeneficialOwnerIndividualDto();
+
+    when(mockPublicPrivateBoPair.getLeft().getName()).thenReturn("Jane Doe");
+
+    Map<String, Pair<PscApi, PrivateBoDataApi>> publicPrivateBoMap = new HashMap<>();
+    publicPrivateBoMap.put(beneficialOwnerIndividualDto.getChipsReference(),
+        mockPublicPrivateBoPair);
+
+    when(mockPublicPrivateBoPair.getLeft().getNationality()).thenReturn("British,French");
+    when(mockPublicPrivateBoPair.getLeft().getNaturesOfControl()).thenReturn(
+        new String[]{"OE_OWNERSHIPOFSHARES_MORETHAN25PERCENT_AS_TRUST"});
+    when(publicPrivateBo.get(beneficialOwnerIndividualDto.getChipsReference())).thenReturn(
+        mockPublicPrivateBoPair);
+
+    var result = beneficialOwnerChangeService.covertBeneficialOwnerIndividualToChange(
+        beneficialOwnerIndividualDto);
+    assertNull(result);
+  }
+
+
+  @Test
+  void testCovertBeneficialOwnerIndividualToChangeIfPairIsNull() {
+    BeneficialOwnerIndividualDto beneficialOwnerIndividualDto = new BeneficialOwnerIndividualDto();
+    beneficialOwnerIndividualDto.setFirstName("John");
+    beneficialOwnerIndividualDto.setLastName("Doe");
+
+    when(mockPublicPrivateBoPair.getLeft().getName()).thenReturn("Jane Doe");
+
+    Map<String, Pair<PscApi, PrivateBoDataApi>> publicPrivateBoMap = new HashMap<>();
```

Comment:
`publicPrivateBoMap` is not currently used anywhere. It may be easier to test this scenario using the `collateBeneficialOwnerChange` method and pass in your created `publicPrivateBO` data?